lib/membrane/compositor_element.ex

```diff
@@ -1,17 +1,34 @@
 defmodule Membrane.VideoCompositor.CompositorElement do
   @moduledoc """
-  The element responsible for placing the first received frame
-  above the other and sending forward buffer with
-  merged frame binary in the payload.
+  The element responsible for composing frames.
+
+  It is capable of operating in one of two modes:
+
+   * offline compositing:
+     The compositor will wait for all videos to have a recent enough frame available and then perform the compositing.
+
+   * live compositing:
+     In this mode, if the compositor will start a timer ticking every spf (seconds per frame). The timer is reset every time a frame is produced.
+     If the compositor doesn't have all frames ready by the time the timer ticks, it will produce a frame anyway, using old frames as fallback in cases when a current frame is not available.
+     If the frames arrive later, they will be dropped.
+
   """
 
   use Membrane.Filter
+  alias Membrane.Buffer
   alias Membrane.RawVideo
   alias Membrane.VideoCompositor.Wgpu
 
   def_options caps: [
-                type: RawVideo,
+                spec: RawVideo.t(),
                 description: "Struct with video width, height, framerate and pixel format."
+              ],
+              live: [
```

Comment:
I know we kept calling this feature "live queueing" but I wonder if real-time isn't a better name for this.